Advanced function usage/task3.js

```
@@ -0,0 +1,65 @@
+import chalk from "chalk";
+
+function divide(numerator, denominator) {
+    if (denominator === 0) {
+        console.log(chalk.red('Ділення на нуль недопустиме!'));

```

you need to throw an Error here